util/chplenv/__init__.py

```diff
@@ -14,6 +15,7 @@
     'chpl_launcher',
     'chpl_llvm',
     'chpl_locale_model',
+    'chpl_machine',
```

Comment:
__init__ was removed in https://github.com/chapel-lang/chapel/pull/11725, please add these to chplenv.py instead